.gitlab-ci.yml

```diff
@@ -55,7 +55,8 @@ variables:
   # Workaround for GitLab intermittent networking
   FF_NETWORK_PER_BUILD: 1
   # Builder image version
-  IMAGE_VERSION: "master"
+  # Use the images created in the mender-test-containers feature branch
+  IMAGE_VERSION: "feature-armhf-native"
```

Comment:
Note to self: this one needs an update after merging https://github.com/mendersoftware/mender-test-containers/pull/545